src/archivist/errors.js

```diff
@@ -1,9 +1,7 @@
 export class InaccessibleContentError extends Error {
   constructor(message) {
-    if (Array.isArray(message)) {
-      message = `\n - ${message.join('\n - ')}`;
-    }
-    super(`The documents cannot be accessed or their contents can not be selected:${message}`);
+    super(`The documents cannot be accessed or their contents can not be selected:${Array.isArray(message) ? `\n - ${message.join('\n - ')}` : message}`);
```

Comment:
> I'm in favour of normalising to always displaying a list, even if there is only one single reason.

Shouldn't we override `toString()` rather than setting `message` to contain all the reasons?